src/ble/att/mod.rs

```diff
@@ -318,38 +331,29 @@ impl<A: Attributes> AttributeServer<A> {
                         handle: AttHandle::NULL,
                     };
 
-                    debug!("ATT Error: {:?}", err);
+                    debug!("ATT error: {:?}", err);
```

Comment:
It would make sense to move this logging into `process_message`, since that would catch *all* ATT errors.

Reply:
Good catch, fixing now